ddl/src/main/resources/templates/spanner/table/create.sqlt

```diff
@@ -0,0 +1,7 @@
+[# th:if="${interleavedTable} == null or ${interleavedTable} == ''"]
```

Comment:
Do we need to handle schema here as well? In the examples above we are checking if schema exists.